tests/test_playbooks/os_default_template.yml

```diff
@@ -10,12 +10,17 @@
   - include: tasks/location.yml
     vars:
       location_state: present
-  - include: tasks/operating_system.yml
+  - include: tasks/operatingsystem.yml
     vars:
-      operating_system_state: present
+      operatingsystem_state: present
   - include: tasks/provisioning_template.yml
     vars:
       provisioning_template_state: present
+  - include: tasks/provisioning_template.yml
```

Comment:
you use two separate tasks here, vs loops later:

```
    loop:
      - provision
      - finish
```

and

```
    loop:
      - Timetravel provision
      - Timetravel finish
```

IMHO we should have one way to do it ;)